### PR TITLE
adding accordion to homepage cms

### PR DESCRIPTION
**Pull request by phillimorland**
still need the design but it's all there
Also need to commit migration once questions are cleared up

### CODE REVIEW

File: meinberlin/apps/cms/models.py [hunk 1]
```diff
@@ -73,6 +73,7 @@ class HomePage(Page):
         ('columns_text', cms_blocks.ColumnsBlock()),
         ('projects', cms_blocks.ProjectsWrapperBlock()),
         ('activities', actions_blocks.PlatformActivityBlock()),
+        ('accordion', cms_blocks.DocsBlock())
     ])
```
Comment (phillimorland) [inline meinberlin/apps/cms/models.py]: Is it ok to call it accordion when block has different name?

File: meinberlin/apps/cms/templates/meinberlin_cms/blocks/docs_block.html [hunk 2]
```diff
@@ -16,8 +19,8 @@ <h2>
 
     <div
         class="accordion__body collapse"
-        id="accordion-{{ index }}-body"
-        aria-labelledby="accordion-{{ index }}-title">
+        id="accordion-{{ identifier }}-body"
+        aria-labelledby="accordion-{{ identifier }}-title">
```
Comment (phillimorland) [inline meinberlin/apps/cms/templates/meinberlin_cms/blocks/docs_block.html]: index wasn't working so added the identifyer stuff from SPD accordion

File: meinberlin/apps/cms/templatetags/menu.py [hunk 3]
```diff
@@ -11,3 +11,8 @@ def get_menu(title):
 
     if menu is not None:
         return menu.items.all()
+
+
+@register.simple_tag()
+def get_identifier(item):
+    return id(item)
```
Comment (phillimorland) [inline meinberlin/apps/cms/templatetags/menu.py]: tried adding this to it's own block_tag.py file but it wouldn't recognise it and listed the acceptable places to add a template tag, did I miss a step?

File: meinberlin/apps/cms/models.py (hunk 1 above)
Comment (MagdaN) [reply meinberlin/apps/cms/models.py]: yes

File: meinberlin/apps/cms/templates/meinberlin_cms/blocks/docs_block.html [hunk 4]
```diff
@@ -1,9 +1,12 @@
 {% load wagtailimages_tags wagtailcore_tags %}
+{% load wagtailcore_tags block_tags %}
```
Comment (fuzzylogic2000) [inline meinberlin/apps/cms/templates/meinberlin_cms/blocks/docs_block.html]: you can and should add all the templatetags to one line. So ehre you just add block _tags to the first line between the brackets and all template tags are loaded.

File: meinberlin/apps/cms/templatetags/menu.py (hunk 3 above)
Comment (fuzzylogic2000) [reply meinberlin/apps/cms/templatetags/menu.py]: You just need to add the tag to a file named whatever you ioad in the template and add that file to a folder templatetags and it should be found. Did that not work?

**Comment by fuzzylogic2000** [comment]
@phillimorland Oh no! You were very smart to leave the migrations out! I forgot about the storefront...

**Pull request merged**